### ISSUE TITLE
any badge shuffle making the game impossible to progress

### ISSUE THREAD

**Issue by minidiamant** [BlastSlimey/PokemonBWAP#3]
It seems that selecting "any badge" for the "shuffle_badge" even with full accessibility can give an impossible generation where the badge you need to get past pinwheel forest aren't accessible, and getting the badges doesn't clear the plasma grunts.

I'll go through what happened to explain what the problem is more clearly;
I set my game in any badge shuffle because someone is playing pokemon emerald in my multiworld. 
I got the quake badge from the first gym, then the jet badge in the second (might be the opposite order I forgot, doesn't matter). i thought it was weird I got badges from my world but I didn't think much more of it cause it happens sometimes with archipelago it gives you items from your own world.
there were other problems I hadn't realised at the time but I'll come back to it.

The plasma museum heist happens, I meet Burgh, and I follow him towards the forest.
There is a wall of plasma guys in the forest talking about the loot sack, and I see Burgh in the distance (
image attached: 

<img width="481" height="504" alt="Image" src="https://github.com/user-attachments/assets/31c50477-17bb-4b86-94fa-fe359a87d7ff" />

). 
I cannot proceed, and I did everything else I knew I could there was nowhere to proceed.
I checked wiki and it says that the plasma grunt are normally there and they leave once you get the basic badge. I ask for the multiworld host to give me the basic badge (which was in the ice gym). they are still there. I ask for the trio badge cause maybe it needs both of them, they are still here.

At that point I check a bunch of things and realise a few problem:
. I cannot sift through my badges in my case (

<img width="961" height="1008" alt="Image" src="https://github.com/user-attachments/assets/2110a8d9-df83-4640-8448-3b3af87ce691" />

)
. lenora is both in the gym, and outside near route 3 searching for team plasma
. the music sounds a bit buggy or im crazy and it's always been like that
. I went back to Striaton to check if talking to the trio guys did a trigger, on the way I notice rippling water, and realised I had not seen any until I got the trio and basic badge.

(my bizhawk is on 2.10 cause I cannot launch the modified nds file on 2.11, btu that's a different issue for a different time I think.)

**Comment by minidiamant** [comment]
mb I didn't see the setup file roadblocks
